test/Microsoft.AspNetCore.Antiforgery.Test/DefaultAntiforgeryTest.cs

```diff
@@ -40,6 +41,28 @@ public class DefaultAntiforgeryTest
         }
 
         [Fact]
+        public async Task ChecksSSL_IsRequestValidAsync_Throws()
+        {
+            // Arrange
+            var httpContext = new DefaultHttpContext();
+
+            var options = new AntiforgeryOptions()
+            {
+                RequireSsl = true
+            };
+
+            var antiforgery = GetAntiforgery(options);
+
+            // Act & Assert
+            var exception = await Assert.ThrowsAsync<InvalidOperationException>(
+                        async () => await antiforgery.IsRequestValidAsync(httpContext));
```

Comment:
Moving a commit comment: Can't this be done w/o `async` or `await` on this line? (Yah know, unnecessary boilerplate.)